### DISCUSSION TITLE
pycuda._driver.memcpy_dtoh_async() did not match C++ signature:memcpy_dtoh_async()

### DISCUSSION THREAD

**Discussion by ywfwyht** [inducer/pycuda#381] [Troubleshooting]
Please delete this issue, I have no problem, now

**Comment by inducer** [comment]
You'll need to provide some more detail to allow someone to meaningfully respond.

**Comment by ZeroE04** [comment]
hi,help me,please!!! how to solute this question? help me please T-T
